hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -102,6 +102,15 @@
             <phase>package</phase>
             <configuration>
               <skip>${maven.shade.skip}</skip>
```

Comment:
If a user directly depend on Ozone and use Ozone's Filesystem without introducing Hadoop's package, can the current Ozone fat-jar provide all necessary dependencies?